src/main/js/main_html5.js

```diff
@@ -1016,6 +1017,15 @@ require("../../../html5-common/js/utils/environment.js");
      * @param {object} event The event from the video
      */
     var raiseTimeUpdate = _.bind(function(event) {
+      // [PBW-6200] - MS Edge will not fire 'playing' or 'canplaythrough' events after seeking.
+      // If playback progresses and 'waitingEventRaised' hasn't been cleared, we fire a
+      // 'buffered' event in order to notify that the video has stopped buffering
```

Comment:
@jpfranco alright, sounds good. Please confirm to see that timeupdates are not sent when seeking again before the video starts playback. Thanks!